src/Libraries/CoreNodeModels/HigherOrder/MapCombineLacing.cs

```diff
@@ -263,11 +263,11 @@ public class Reduce : VariableInputNode
 
         public Reduce()
         {
-            InPortData.Add(new PortData("reductor", Resources.ReducePortDataReductorToolTip));
-            InPortData.Add(new PortData("seed", Resources.ReducePortDataSeedToolTip));
-            InPortData.Add(new PortData("list1", Resources.PortDataListToolTip + " #1"));
+            InPorts.Add(new PortModel(PortType.Input, this, new PortData("reductor", Resources.ReducePortDataReductorToolTip)));
```

Comment:
Nothing. This design is not great and needs to be cleaned up, but I didn't want to do it as part of this PR. We should remove `PortType` because it doesn't have any real meaning. Whether a port is an "input" or an "output" can be determined by whether it's in the `InPorts` or the `OutPorts` collections respectively.